modules/apps/portal-transaction/portal-transaction-test/src/testIntegration/java/com/liferay/portal/transaction/test/TransactionInvokerUtilTest.java

```diff
@@ -121,6 +135,67 @@ public void testRollback() {
 		}
 	}
 
+	@Ignore
+	@Test
+	public void testTransactionRollbackKeepsFileInStore() throws Throwable {
+		byte[] bytes = TestDataConstants.TEST_BYTE_ARRAY;
+
+		InputStream inputStream = new ByteArrayInputStream(bytes);
+
```

Comment:
Hey @kyle-miho, 
Can you inline the `new ByteArrayInputStream(bytes)` statement in the _`dlFileEntryLocalService.addFileEntry(...)` call? It is just used there. Thx